File: AcupayEmployee/Core Forms/Government/SocialSecurityBracket.vb
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.ComponentModel.DataAnnotations

Namespace Global.PayrollSys

    <Table("paysocialsecurity")>
    Public Class SocialSecurityBracket

        <Key>
        Public Property RowID As Integer?

        Public Property Created As Date

        Public Property CreatedBy As Integer?

        Public Property LastUpd As Date

        Public Property LastUpdBy As Integer?

        Public Property RangeFromAmount As Decimal

        Public Property RangeToAmount As Decimal

        Public Property MonthlySalaryCredit As Decimal

        Public Property EmployeeContributionAmount As Decimal

        Public Property EmployerContributionAmount As Decimal

        Public Property EmployeeECAmount As Decimal

        Public Property HiddenData As Boolean

    End Class

End Namespace

File: Accupay.Data/Entities/SocialSecurityBracket.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace AccuPay.Data.Entities
{
    [Table("paysocialsecurity")]
    public class SocialSecurityBracket
    {
        [Key]
        public int? RowID { get; set; }

        public DateTime Created { get; set; }

        public int? CreatedBy { get; set; }

        public DateTime? LastUpd { get; set; }

        public int? LastUpdBy { get; set; }

        public decimal RangeFromAmount { get; set; }

        public decimal RangeToAmount { get; set; }

        public decimal MonthlySalaryCredit { get; set; }

        public decimal EmployeeContributionAmount { get; set; }

        public decimal EmployerContributionAmount { get; set; }

        public decimal EmployeeECAmount { get; set; }

        public bool HiddenData { get; set; }

        public DateTime EffectiveDateFrom { get; set; }

        public DateTime EffectiveDateTo { get; set; }
    }
}